Tests/aweXpect.Reflection.Tests/ThatType.DoesNotInheritFrom.Tests.cs

```diff
@@ -0,0 +1,39 @@
+using Xunit.Sdk;
+
+namespace aweXpect.Reflection.Tests;
+
+public sealed partial class ThatType
+{
+	public sealed class DoesNotInheritFrom
+	{
+		public sealed class Tests
```

Comment:
Rename to `GenericTests` and also add type tests accordingly (with the grandchild-inheritance and inheritance of an interface tests).